wallet/src/de/schildbach/wallet/wallofcoins/WOCConstants.java

```diff
@@ -0,0 +1,34 @@
+package de.schildbach.wallet.wallofcoins;
+
+/**
+ * Created by WOC on 27-Jan-18.
+ * Copyright (c) 2018 WOC Limited. All rights reserved.
+ */
+
+public final class WOCConstants {
+    // REST FUll API KEY Constant
+    public static final String KEY_HEADER_AUTH_TOKEN = "X-Coins-Api-Token";
+    public static final String KEY_HEADER_PUBLISHER_ID = "X-Coins-Publisher";
```

Comment:
Where is this value used to pass as a HEADER to the API? Provide file name and line number.

Reply:
@rgenito  We are using this constants file as key and other constant parameter in BuyDashFragment,java file